clients/command-kafka/src/main/java/org/eclipse/hono/client/command/kafka/KafkaBasedInternalCommandConsumer.java

```diff
@@ -196,13 +229,31 @@ public final KafkaBasedInternalCommandConsumer setMetricsSupport(final KafkaClie
         adminClient.createTopics(List.of(newTopic))
                 .all()
                 .whenComplete((v, ex) -> {
-                    context.runOnContext(v1 -> Optional.ofNullable(ex).ifPresentOrElse(promise::fail, promise::complete));
+                    context.runOnContext(v1 -> Optional.ofNullable(ex)
+                            .filter(e -> !(e instanceof TopicExistsException))
+                            .ifPresentOrElse(promise::fail, promise::complete));
                 });
         return promise.future()
                 .onSuccess(v -> LOG.debug("created topic [{}]", topicName))
                 .onFailure(thr -> LOG.error("error creating topic [{}]", topicName, thr));
     }
 
+    private Future<Void> createTopicWithRetries() {
+        final Promise<Void> createTopicRetryPromise = Promise.promise();
+        // Retry at specified interval until the internal command topic is successfully created
+        vertx.setPeriodic(CREATE_TOPIC_RETRY_INTERVAL, id -> {
+            if (retryCreateTopic.compareAndSet(true, false)) {
+                createTopic()
```

Comment:
This means, there is always a one second delay before the first attempt to create the topic is made.
It would be better if the first attempt is made directly.